src/interface_adapter/vote_out/VoteOutController.java

```diff
@@ -7,7 +7,6 @@ public class VoteOutController {
     final VoteOutInputBoundary userVoteOutUseCaseInteractor;
 
     public VoteOutController(VoteOutInputBoundary userVoteOutUseCaseInteractor) {
-        // TODO: check if the input boundary should be passed in as a parameter later (they did that in week5ca)
```

Comment:
Note for Ethan: we do indeed pass it in since something concrete (i.e. dirty) must be passed in at some point)